java/client/src/main/java/glide/api/commands/StringBaseCommands.java

```diff
@@ -156,6 +156,25 @@ public interface StringBaseCommands {
      */
     CompletableFuture<String> mset(Map<String, String> keyValueMap);
 
+    /**
+     * Sets multiple keys to multiple values in a single operation. Performs not operation at all even
+     * if just a single key already exists.
+     *
+     * @apiNote When in cluster mode, the command may route to multiple nodes when keys in <code>
+     *     keyValueMap</code> map to different hash slots.
+     * @see <a href="https://redis.io/commands/msetnx/">redis.io</a> for details.
+     * @param keyValueMap A key-value map consisting of keys and their respective values to set.
+     * @return <code>true</code> if all keys were set, <code>false</code> if no key was set.
+     * @example
+     *     <pre>{@code
+     * Boolean result = client.msetnx(Map.of("key1", "value1", "key2", "value2"}).get();
+     * assertTrue(result);
+     * Boolean result = client.msetnx(Map.of("key1", "value1", "key2", "value2"}).get();
+     * assertFalse(result);
```

Comment:
this second example really adds no value... can you remove it?

Reply:
Was thinking this should demonstrate the not adding to existing keys behavior, will remove.